libs/base/Control/Monad/Maybe.idr

```diff
@@ -0,0 +1,39 @@
+module Control.Monad.Maybe
+
+import Control.Monad.Trans
+
+%access public export
+
+data MaybeT : (m : Type -> Type) -> (a : Type) -> Type where
+    MkMaybeT : m (Maybe a) -> MaybeT m a
+
+runMaybeT : MaybeT m a -> m (Maybe a)
+runMaybeT (MkMaybeT v) = v
```

Comment:
Instead of declaring `data` and a "getter" separately, you can write 
```
record MaybeT (m : Type -> Type) (a : Type) where
  constructor MkMaybeT
  runMaybeT : m (Maybe a)
```